pkgs/development/python-modules/minio/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
+, urllib3, python-dateutil , pytz}:
+
+buildPythonPackage rec {
+  pname = "minio";
+  version = "4.0.11";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
```

Comment:
I see no tests in the source distribution.
Please fetch this from the [github repo](https://github.com/minio/minio-py) using `fetchFromGitHub`.

You'll also need to add the [testing requirements](https://github.com/minio/minio-py/blob/bd675fcca2b2e83f2f6b0a1de504a1b0abdb5ad0/setup.py#L49) to `checkInputs`.

Reply:
the tests run and passed without it?